client/.storybook/.babelrc

```diff
@@ -0,0 +1,7 @@
+{
+	"presets": ["es2015", "react"],
```

Comment:
@oahmedqureshi looks like an indentation issue

Reply:
I'm not committing these extra spaces